tests/chassischain/test_chassis_chain_support_2.py

```diff
@@ -0,0 +1,126 @@
+def test_chassis_chain_support_2(api, b2b_raw_config, utils):
+
+    # Chassischain configuration
+    ixnconfig = api.ixnetworkconfig
```

Comment:
we need to add validation that those are setting properly through IxNetwork. This is RestPy IxNetwork handler 'api._ixnetwork`. So, you will get chassis using `api._ixnetwork.AvailableHardware.Chassis`. Please take a look in this test for reference `tests\vxlan\test_manual_gateway_mac.py` 


These tests probably not run in regular pipeline.